.gitignore

```diff
@@ -336,10 +336,20 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
 /src/Simulation/Simulators.Tests/TestProjects/QSharpExe/built
+/src/Simulation/Simulators.Tests/TestProjects/QirExe/qir
 /src/Simulation/Simulators.Tests/TestProjects/TargetedExe/built
+
 dbw_test
 
+# Controller test artifacts
+/src/Qir/Controller/test-artifacts/
+/src/Qir/Controller/src
```

Comment:
1. `/src/Qir/Controller/src` is a file or directory? (why one line above ends with a slash `/`, but this line ends not with a slash?)
2. It seems rather suspicious that `git` is instructed to ignore a file/directory named **_`src`_**. If the file/directory named `src` is generated during the build, then may be it makes sense to rename `src` to something more evident as a temporary file/directory?

Reply:
1. `/src/Qir/Controller/src` was a directory. It was an error that it didn't end with a slash `/`.
2. The reason why `git` was instructed to ignore a directory named `src` was because the controller (which is no longer hosted in this repo) produced some artifacts (namely the generated C++ entry-point driver file) as part of the process of building a QIR executable.

Since the controller is no longer hosted in this repo, I removed these lines.

Good catch! Thanks Robin.